### DISCUSSION TITLE
GitHub Co-Pilot access for student developers

### DISCUSSION THREAD

**Discussion by atomisadev** [community/community#28494] [Copilot Conversations] [answered]
Hi, I am currently a student enrolled in the Developer Pack.

Currently, Copilot subscriptions are not available for students (by the time of writing this, Copilot is out of beta and is released commercially). Will it be possible sometime in the near future for students to have Copilot subscriptions part of the Developer Pack?

Thanks,
Atom

**Comment by davecheney** [comment]
Copilot is available without charge for anyone with an active student coupon.

**Comment by davecheney** [comment, marked as answer]
@atomdevelops I have confirmed you have an active `student-2022` coupon, and you have access to copilot.

**Comment by Odettas** [comment]
Hi, i need a help for registering for student developer pack because i can't upload my application even tho it already has the requirement in it

**Reply by davecheney** [reply]
@Odettas i'm sorry to hear that you are having difficulty. Please contact support who should be able to assist you.

**Reply by Love2Bit** [reply]
I already contact the education github mail buat my ticket denied this the same person as odettas

**Reply by davecheney** [reply]
I'm sorry to hear this. I am not able to assist you as I do not have any visibility into the education coupon process.

**Comment by thebozturk** [comment]
Hi, I have a student pack but i can't use co-pilot. How can i solve it?

**Reply by davecheney** [reply]
@thebozturk im sorry you are experiencing difficulties. Can you please tell me what you’ve tried, and what happened when you tried?

**Reply by thebozturk** [reply]
I'm getting this message. I used Technical Preview before but I have a student pack and github pro. I think I can use co-pilot. 
![resim](https://user-images.githubusercontent.com/70200551/211221082-d07ff096-051e-4efb-8dda-02183c86ad06.png)

**Reply by SlimGee** [reply]
I'm having the same issue. I was in the technical preview program as well

**Reply by davecheney** [reply]
@thebozturk @SlimGee please contact support https://support.github.com/contact

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by kayibea** [comment]
Me too i got the same things
![Screenshot from 2023-01-09 01-01-16](https://user-images.githubusercontent.com/94308564/211225488-11acabd0-925c-4ecc-ae0c-dc8a5db63296.png)

**Reply by Iloveyousomuchben** [reply]
me neither

**Reply by ushiii111798** [reply]
Same here

**Reply by ni920** [reply]
same here

**Reply by sharon2000** [reply]
same here.

**Reply by marekasf** [reply]
Same problem for me.

**Reply by momalekabid** [reply]
Likewise!

**Comment by sharon2000** [comment]
I am also facing the same issue. My student developer pack is active. But when i try to enable github copilot am getting option to pay 10$ for it. I didnt have this issue before. But my account was asked to re-verify student status and was approved.
However, now am not able to active Github copilot. Please help!

**Comment by karkraeg** [comment]
Same here, student developer pack is active but my access to Co-Pilot was revoked.

**Reply by karkraeg** [reply]
re-enabled, after that it works fine again.
